src/gei/GithubApiFactory.cs

```diff
@@ -23,6 +22,16 @@ GithubApi ISourceGithubApiFactory.Create()
             return new GithubApi(githubClient);
         }
 
+        GithubApi ISourceGithubApiFactory.CreateDefaultClient()
+        {
+            var githubPat = _environmentVariableProvider.SourceGithubPersonalAccessToken();
+
+            #pragma warning disable CA2000 // We don't want to dispose the handler until the client is disposed
+            var client = new HttpClientFactory().CreateClient("OctoShift");
+            var githubClient = new GithubClient(_octoLogger, client, githubPat);
+            return new GithubApi(githubClient);
+        }
+
```

Comment:
What is the purpose of this factory method? it seems like it is exactly creating an Api corresponding the the source PAT. 
Also we use DI, so there is no need to have `var client = new HttpClientFactory().CreateClient("OctoShift");` b/c it defies the reason why we use DI in the first place!

Reply:
We have this factory so we can manipulate the client for specific kinds of http client connections that we might want. For example, for ghes, we will want to not follow redirects and allow non-ssl signed client connections